File: image-coffee-utils-crop/Crop/Adapter/In/Response/FailureResponse.cs
namespace image_coffee_utils_crop.Crop.Adapter.In.Response
{
    /// <summary>
    /// Failure response.
    /// </summary>
    /// <param name="message">The failure message</param>
    public class FailureResponse(string message)
    {
        /// <summary>
        /// The failure message.
        /// </summary>
        public string Message { get; } = message;
    }
}

File: image-coffee-utils-java-shared/src/main/java/xyz/cupscoffee/imagecoffeeutils/shared/application/response/FailureResponse.java
package xyz.cupscoffee.imagecoffeeutils.shared.application.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

/**
 * Represents a basic failure response.
 */
@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public final class FailureResponse {
    private String message;
}
